README.md

```diff
@@ -38,49 +34,23 @@ command.
 |  `MYSQL_DATABASE`      | Database name                             |
 |  `MYSQL_ROOT_PASSWORD` | Password for the root user (optional)     |
 
-You can also set following mount points by passing `-v /host:/container`
-flag to docker.
+You can also set following mount points by passing `-v /host:/container` flag to docker.
 
-|  Volume mount point | Description          |
-| :------------------ | -------------------- |
-|  `/var/lib/mysql`   | MySQL data directory |
+|  Volume mount point      | Description          |
+| :----------------------- | -------------------- |
+|  `/var/lib/mysql/data`   | MySQL data directory |
 
 ## Usage
 
-We will assume that you are using the `openshift/mysql-55-centos7`
-image. Supposing that you want to set only mandatory required environment
-variables and store the database on in the `/home/user/database`
-directory on the host filesystem, you need to execute the following
-command:
+We will assume that you are using the `openshift/mysql-55-centos7` image. Supposing that you want to set only mandatory environment variables and not store the database directory on the host filesystem, you need to execute the following command:
 
 ```console
-docker run -d -e MYSQL_USER=<user> -e MYSQL_PASSWORD=<password> -e MYSQL_DATABASE=<database> -v /home/user/database:/var/lib/mysql openshift/mysql-55-centos7
+docker run -d --name mysql_database -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db -p 3306:3306 openshift/mysql-55-centos7
 ```
 
-If the database directory is not initialized, the entrypoint script will
-first run `mysql_install_db` and setup necessary database users and
-passwords. After the database is initialized, or if it was already
-present, `mysqld` is executed and will run as PID 1. You can stop the
-detached container by running `docker stop <CONTAINER ID>`.
+This will create a container named `mysql_database` running MySQL with database `db` and user with credentials `user:pass`. Port 3306 will be exposed and mapped to host. If you want your database to be persistent across container executions, also add a `-v /host/db/path:/var/lib/mysql/data` argument. This is going to be the MySQL data directory.
 
-### MySQL root user
-The root user has no password set by default, only allowing local
-connections. You can set it by setting `MYSQL_ROOT_PASSWORD` environment
-variable when initializing your container. This will allow you to login
-to the root account remotely. Local connections will still not require
-password.
-
-## Software Collections
```

Comment:
it's probably still worth mentioning that the image is based on SCL.
1) it's a good plug
2) for people who docker exec into it, it may help them understand the directory structure better.
3) for people who docker exec commands directly (not running /bin/sh) they might need to know this.